modules/rest-util/src/blaze/middleware/fhir/output.clj

```diff
@@ -53,6 +65,15 @@
   (cond-> response body (-> (update :body generate-xml)
                             (ring/content-type content-type))))
 
+(defn- binary-content-type [body]
+  (when body
```

Comment:
the body isn't nil here, because you check for nil already in  `encode-response-binary`. even than both the function `:contentType` and `fhir-type/value` will accept nil and return nil

Reply:
I was just being unnecessarily defensive here, ha ha.